### PR TITLE
added versions to user settings page

### PR DESCRIPTION
**Pull request by MitskevichVS**
#### Screenshots

Screen Shot 2020-02-19 at 11 16 40
Screen Shot 2020-02-19 at 11 16 58

### CODE REVIEW

File: client/app/_user/user.controller.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ const UserProfileController = function UserProfileController($mdDialog, UserServ
         goToState,
 
         get currentUser() { return UserService.currentUser; },
-        get serviceUrl() { return $rootScope.version && $rootScope.version.service_url || ''; }
+        get serviceUrl() { return $rootScope.version && $rootScope.version.service_url || ''; },
+        get appVersions() { return {server: $rootScope.version.service, client: $rootScope.version.client} || {}}
```
Comment (danymill) [inline client/app/_user/user.controller.js]: incorrect code

File: client/app/_user/profile.html [hunk 2]
```diff
@@ -11,23 +11,30 @@
...
+            <div layout="row" flex="auto">
+                <div class="profile-img">
+                    <profile-photo ng-model="$ctrl.currentUser.photoURL" size="48">
+                        <div class="profile-img-content">
+                            <i class="material-icons" ng-click="$ctrl.showUploadImageDialog($event)">add_a_photo</i>
+                        </div>
+                    </profile-photo>
+                </div>
+                <div class="profile-wrapper" flex="100">
+                    <div>
```
Comment (danymill) [inline client/app/_user/profile.html]: please add a class if you need this `div`

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/_user/user.controller.js (hunk 1 above)
Comment (MitskevichVS) [reply client/app/_user/user.controller.js]: fixed

File: client/app/_user/profile.html (hunk 2 above)
Comment (MitskevichVS) [reply client/app/_user/profile.html]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**